index.js

```diff
@@ -3,19 +3,32 @@
 const url = require('url')
 const fp = require('fastify-plugin')
 const zipkin = require('zipkin')
+const assert = require('assert')
 
 const Some = zipkin.option.Some
 const None = zipkin.option.None
 const Instrumentation = zipkin.Instrumentation
 
 function zipkinPlugin (fastify, opts, next) {
-  const tracer = opts.tracer
-  if (tracer == null) return next(new Error('No tracer specified'))
+  assert(opts.serviceName, 'serviceName option should not be empty')
+  const { Tracer, BatchRecorder, jsonEncoder: { JSON_V2 } } = require('zipkin')
+  const { HttpLogger } = require('zipkin-transport-http')
+  const CLSContext = require('zipkin-context-cls')
```

Comment:
Can you move these requires to the top of the file?

Reply:
Done! 👍